contracts/eoslib/types.hpp

```diff
@@ -89,4 +89,71 @@ namespace  eosio {
    template<typename T> struct remove_reference<const T&> { typedef T type; };
    ///@}
 
+    /**
```

Comment:
Please move this to a file specifically for compliance with standard lib features.   And add it to its own Doxygen group whose description describes what these do, _not_ why they are needed.  For instance, "support for compile time C++ standard features"

Reply:
how about eos/contracts/eoslib/type_traits.hpp ?

Reply:
works for me.